cli/ops/cephadm.py

```diff
@@ -182,3 +184,35 @@ def bootstrap(
         raise OperationFailedError("Failed to bootstrap cluster")
 
     return True
+
+
+def is_prometheus_enabled(installer):
+    """Checks if prometheus is enabled
+    Args:
+        installer (CephInstallerNode): Ceph installer node
+    """
+    if get_mon_stack_status(installer, "prometheus") == "on":
+        return False
+    return True
+
+
+def is_dashboard_enabled(installer):
+    """Checks if dashboard is enabled
+    Args:
+        installer (CephInstallerNode): Ceph installer node
+    """
+    if not get_mon_stack_status(installer, "dashboard") == "on":
+        return False
+    return True
+
+
+def get_mon_stack_status(installer, mon_stack):
```

Comment:
Change this to `get_module_status`, update veriables accordingly

Reply:
Updated.